### ISSUE TITLE
[feature request] remove notification dot from silent notifications

### ISSUE THREAD

**Issue by gilcu3** [MM2-0/Kvaesitso#266]
Allow hiding notification dot for silent notifications. For example the FOSS version of telegram keeps a notification for the push service, and I cannot hide the corresponding badge from the favorite apps in Kvaesitso.

**Comment by Sir-Photch** [comment]
IIRC you can just disable the Telegram-notification for the push service. This should make that disappear.

**Comment by gilcu3** [comment]
> IIRC you can just disable the Telegram-notification for the push service. This should make that disappear.

True, although I prefer to have it silent. In other FOSS launchers the dot notification doesn't appear so there should be a way to implement it. Still, thanks for the advice

**Comment by gilcu3** [comment]
it seems that this was recently fixed!

**Comment by gilcu3** [comment]
Sorry for reopening, but suddenly this began happening again in the current version `1.29.1` from fdroid
